### DISCUSSION TITLE
Add `turbo exec` command

### DISCUSSION THREAD

**Discussion by eric-burel** [vercel/turborepo#375] [Ideas]
### Describe the feature you'd like to request

Hi, I'd like to add full-fledged commands to Turborepo pipelines, instead of script names.
For example, here is my local publish script with yalc: `"publish:local": "lerna exec yalc push"`. It is dependent on the "build" pipeline.

I need the "exec" to avoid the need to define a "publish:local" script in each of my underlying package, which would be unefficient. Turborepo documents only the "run" function, is it possible to have something similar to "exec"?

### Describe the solution you'd like

Documentation for replacing "lerna exec" by "turbo exec" in https://turborepo.org/docs/guides/migrate-from-lerna

### Describe alternatives you've considered

Not using Turborepo so I build manually and then publish.

**Comment by jaredpalmer** [comment]
I dig it. The ability to quickly have commands that apply to all packages but are defined in the root or pipeline will make migration a lot easier too.

**Reply by ma-hiro** [reply]
Is there currently a method provided to do this?
I prefer to keep the project's package.json as simple as possible. Thanks.

**Reply by TheSonOfThomp** [reply]
Is there any progress on this? I still have to keep `lerna` installed to execute some tasks. Would love to be able to fully migrate to turbo

**Comment by mehulkar** [comment]
This is tracked in https://github.com/vercel/turbo/issues/4549
